tests/conftest.py

```diff
@@ -77,6 +77,12 @@ async def query(
             self.waiting.set()
             await self.pause.wait()
             self.waiting.clear()
+
+        if isinstance(expression, exp.Describe):
```

Comment:
Do we need this?
I'm not sure what the assert is doing below. It must be intended for the `execute` call.